javav2/example_code/s3/src/main/java/com/example/s3/S3AsyncOps.java

```diff
@@ -36,22 +28,19 @@
 
      public static void main(String[] args) {
 
-
          final String USAGE = "\n" +
                  "Usage:\n" +
-                 "    S3AsyncOps <bucketname> <objectname> <path>\n\n" +
+                 "    S3AsyncOps <bucketName> <key> <path>\n\n" +
                  "Where:\n" +
-                 "    bucketname - the name of the bucket (i.e., bucket1)\n\n" +
-                 "    objectname - the name of the object (i.e., book.pdf)\n" +
-                 "    path - the local path to the file (i.e., C:\\AWS\\book.pdf)\n" +
-                 "Example:\n" +
-                 "    bucket1 book.pdf  C:\\AWS\\book.pdf\n";
+                 "    bucketName - the name of the Amazon S3 bucket (for example, bucket1). \n\n" +
+                 "    key - the name of the object (for example, book.pdf). \n" +
+                 "    path - the local path to the file (for example, C:\\AWS\\book.pdf). \n" ;

```

Comment:
'C:\\AWS\\book.pdf' here, but 'C:/AWS/book.pdf' in some code comments in earlier files?

Reply:
fixed